README.md

```diff
@@ -1,284 +1,126 @@
+- In the _Environment variables (advanced)_ section add a new variable named `NODE_VERSION` set to `16` or greater (`18` is not supported yet, See [Build Image Update Discussion](https://github.com/cloudflare/pages-build-image/discussions/1)).
+- Click on _Save and Deploy_ to start the deployment (this first deployment won't be fully functional as the next step is also necessary)
+- Go to the Pages project settings page (_Settings_ > _Functions_ > _Compatibility Flags_), add the `nodejs_compat` for both the production and preview and make sure that the **Compatibility Date** for both production and preview is set to at least `2022-11-30`.
 
-```diff
-/** @type {import('next').NextConfig} */
-const nextConfig = {
-  experimental: {
-    runtime: "experimental-edge",
-+   appDir: true,
-  },
-  reactStrictMode: true,
-  swcMinify: true,
-};
+> If you don't want to set up a git repository, you can build your `_worker.js` file (as indicated in [Local Development](#local-development)) and publish your application manually via the [wrangler's pages publish command](https://developers.cloudflare.com/workers/wrangler/commands/#publish-1) instead (but you'll still need to set the `NODE_VERSION` environment variable and the `nodejs_compat` flag for your project in the Cloudflare dashboard).
 
-module.exports = nextConfig;
-```
+## Local development
 
-If you're following the [Next.js 12 → 13 Upgrade Guide](https://beta.nextjs.org/docs/upgrade-guide#step-4-migrating-pages), delete any `./pages/_app.tsx` and `./pages/index.tsx` files and replace with `./app/layout.tsx` and `./app/page.tsx`:
-
-```typescript
-// ./app/layout.tsx
-import '../styles/globals.css';
-import { FC } from 'react';
-
-const RootLayout: FC<{
-	children: React.ReactNode;
-}> = ({
-	// Layouts must accept a children prop.
-	// This will be populated with nested layouts or pages
-	children,
-}) => {
-	return (
-		<html lang="en">
-			<body>{children}</body>
-		</html>
-	);
-};
+To locally run the CLI simply run:
 
-export default RootLayout;
+```sh
+npx @cloudflare/next-on-pages
 ```
 
-```typescript
-// ./app/page.tsx
-import { FC } from 'react';
-import styles from '../styles/Home.module.css';
-
-const Home = async (): Promise<ReturnType<FC>> => {
-	const uuid = await fetch('https://uuid.rocks/plain').then(
-		async response => await response.text()
-	);
-
-	return (
-		<div className={styles.container}>
-			<main className={styles.main}>
-				<h1 className={styles.title}>
-					Welcome to <a href="https://nextjs.org">Next.js!</a>
-				</h1>
-
-				<p className={styles.description}>
-					Get started by editing{' '}
-					<code className={styles.code}>pages/index.tsx</code>
-				</p>
-
-				<p className={styles.description}>
-					Here&apos;s a server-side UUID:
-					<code className={styles.code}>{uuid}</code>
-				</p>
-			</main>
-		</div>
-	);
-};
+This command will build your Next.js application and bundle a `_worker.js` file for you to use.
```

Comment:
I don't think most people know about (or care about) `_worker.js`. Instead, we should frame this as:

````suggestion
This command will build your Next.js application and produce a `.vercel/output/static` directory which you can then use with Wrangler:

```sh
npx wrangler pages dev .vercel/output/static --compatibility-flag=nodejs_compat
```
````